cmd/launcher/launcher.go

```diff
@@ -197,9 +197,9 @@ func runLauncher(ctx context.Context, cancel func(), multiSlogger, systemMultiSl
 	// Generate a new run ID
 	newRunID := k.GetRunID()
 
-	// Apply the run ID to both logger and slogger
+	// Apply the run ID to logger which is later used in logshipper
+	// slogger run_id is set in the knapsack GetRunID()
```

Comment:
Approved, but I wonder if it's easier to just put `k.GetRunID()` here, and not create the `newRunID`

Reply:
We still need the run_id for the other logger that is used in the logshipper since that doesn't use knapsack. Since Becca's comment that the setup should still be in launcher I'll leave things as is and close this PR.